### DISCUSSION TITLE
@graphql-codegen/add question

### DISCUSSION THREAD

**Discussion by Burnett2k** [dotansimha/graphql-code-generator#7456] [Q&A] [answered]
I'm considering using the /add plugin to share place a few files in our generated file along with the typical typescript types generated from our `gql` tags. The documentation on the `content` config option is confusing and there doesn't appear to be an example showing how to include a file path. The docs say:

> The actual content you wish to add, either a string or array of strings. You can also specify a path to a local file and the content if it will be loaded by codegen.

Do you mind providing an example of how to pass in a path to a file? I've tried a few things and it always seems to treat it as a string.

**Comment by dimaMachina** [comment, marked as answer]
I think documentation is incorrect, I looked at [the source code](https://github.com/dotansimha/graphql-code-generator/blob/master/packages/plugins/other/add/src/index.ts) of `add` plugin and there is no logic that handles loading from path.

**Reply by dimaMachina** [reply]
@charlypoly what do you think?

**Reply by charlypoly** [reply]
@Burnett2k @B2o5T I confirm that the documentation is incorrect on this point

Only string of actual snippet of code can be passed to the `content` property
